src/integrations/admin/social-templates-integration.php

```diff
@@ -204,11 +205,20 @@ protected function build_social_fields( Yoast_Form $yform, $identifier, $page_ty
 				'label_title'             => \__( 'Social title', 'wordpress-seo' ),
 				'label_description'       => \__( 'Social description', 'wordpress-seo' ),
 				'description_placeholder' => \__( 'Modify your social description by editing it right here.', 'wordpress-seo' ),
-				'is_disabled'             => ! $is_premium,
+				'is_disabled'             => ! $is_form_enabled,
 			]
 		);
 		$editor->render();
 
+		if ( $is_premium && ! $is_premium_16_5 ) {
+			echo '<div class="yoast-settings-section-upsell">';
+
+			echo '<p>'
+			. \esc_html__( 'Unlock by updating Yoast SEO Premium or letting your admin update it', 'wordpress-seo' )
```

Comment:
Usually, we don't want the plugin names to be translatable. We should use a sprintf to move "Yoast SEO Premi" out of hte translatable string.